### PR TITLE
Update devchains

### LINKED ISSUE #515: Update development chain setup
Right now we use wasmd for all launchpad testing. When we upgrade it, we loose basic launchpad testing. Let's upgrade the local chain setup to something like this:

- [x] Rename chain `wasmd` to `launchpad`, including all testing code referring to it (ports 1317/26657)
- [x] Keep simapp but change ports (ports 1318/26658, i.e. default+1)
- [ ] Add new chain stargate based `wasmd` (ports 1319/26659, i.e. default+2)
- [ ] Update ports table in HACKING.md

### PR DESCRIPTION
**Pull request by willclarktech**
Part 1 of #515

### CODE REVIEW

File: scripts/simapp/start.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 
 # Please keep this in sync with the Ports overview in HACKING.md
 TENDERMINT_PORT_GUEST="26657"
-TENDERMINT_PORT_HOST="26657"
+TENDERMINT_PORT_HOST="26658"
```
Comment (webmaster128) [inline scripts/simapp/start.sh:7]: Can you update the ports list in HACKING.md accordingly?

**Review by webmaster128 — COMMENTED** [review]
👌

File: scripts/simapp/start.sh (hunk 1 above)
Comment (willclarktech) [reply scripts/simapp/start.sh:7]: I thought I did! What did I leave out?

Comment (willclarktech) [reply scripts/simapp/start.sh:7]: Oh, just didn't push the latest change.

**Review by webmaster128 — APPROVED** [review]
All there now, thanks

**Pull request merged**